### ISSUE TITLE
Not possible to build on Windows

### ISSUE THREAD

**Issue by erikarenhill** [leandrowd/react-responsive-carousel#440]
Currently it's not possible to build the project on Windows.

Adding a PR (https://github.com/leandrowd/react-responsive-carousel/pull/441) with a suggested solution using cross-env and babel parameter --extension '.ts'

**Comment by erikarenhill** [comment]
https://github.com/leandrowd/react-responsive-carousel/pull/441

**Comment by ludem** [comment]
Hi, I got an error on IE11 because of the following lines in Thumbs.js, where shorthand property names (ES2015) are used (not supported in IE11):

```
 _this.setState(function (_state, props) {
        return {
          itemSize,
          visibleItems,
          firstItem: showArrows ? _this.getFirstItem(props.selectedItem) : 0,
          lastPosition,
          showArrows
        };
      });
```

**Comment by erikarenhill** [comment]
> Hi, I got an error on IE11 because of the following lines in Thumbs.js, where shorthand property names (ES2015) are used (not supported in IE11):
> 
> ```
>  _this.setState(function (_state, props) {
>         return {
>           itemSize,
>           visibleItems,
>           firstItem: showArrows ? _this.getFirstItem(props.selectedItem) : 0,
>           lastPosition,
>           showArrows
>         };
>       });
> ```
@ludem I think that's more related to https://github.com/leandrowd/react-responsive-carousel/issues/439 ! And if configuring babelrc to allow IE11 it should work as well with this. But I'm waiting for a response from @leandrowd to see if IE11 should be supported before making a PR

**Comment by ludem** [comment]
@erikarenhill You're right, I meant to comment the other thread!

**Comment by leandrowd** [comment]
I've removed the env var so please pull and check if you can build the new version on windows.

**Comment by erikarenhill** [comment]
> I've removed the env var so please pull and check if you can build the new version on windows.

I tried it but the output is just "Successfully compiled 0 files with Babel." meanwhile the output from my PR is showing 6 files. Will try and find why later today (running yarn run lib:build-es)

The style compilation does not run on Windows either due to `mkdir` reporting incorrect syntax, probably due to the slashes (/ while Windows like \\ better). Maybe mkdirp is an option? (https://www.npmjs.com/package/mkdirp), I tried it and that seem to work.

**Comment by ludem** [comment]
If it can help, my issue is totally resolved by the 3.2.7. 
Thank you.

**Comment by erikarenhill** [comment]
@leandrowd For babel to work on windows, the .ts,.tsx seems to must be without the single quotes: https://github.com/babel/babel/issues/9803#issuecomment-505435797

I tested this and after removing the single quotes it compiles. Also still must replace mkdir for mkdirp or similiar solution to have it work in Windows.
